csrc/host_ir/host_ir.cpp

```diff
@@ -122,9 +122,13 @@ bool PostOnStream::sameAs(const Statement* other) const {
 LaunchKernel::LaunchKernel(
     IrBuilderPasskey passkey,
     int64_t hic_executor_index,
+    const LaunchParams& launch_constraints,
+    const CompileParams& compile_params,
     const std::vector<Val*>& inputs,
     const std::vector<Val*>& outputs)
-    : Expr(passkey, inputs, outputs, {}) {
+    : Expr(passkey, inputs, outputs, {}),
+      launch_constraints_(launch_constraints),
+      compile_params_(compile_params) {
   addDataAttribute(hic_executor_index);
```

Comment:
We should also add launch_constraints and compile_params as data attribute. Or you have to define a custom cloner for cloning to work correctly.